snapcraft/tests/pluginhandler/test_scriptlets.py

```diff
@@ -24,6 +24,16 @@
 
 class ScriptletTestCase(tests.TestCase):
 
+    def test_run_prepare_scriptlet(self):
+        handler = mocks.loadplugin(
```

Comment:
I know I reviewed the mocks PR, but technically this is a fake: http://xunitpatterns.com/Fake%20Object.html
(testxplaning)